libres/lib/enkf/obs_data.cpp

```diff
@@ -209,20 +209,6 @@ static void obs_block_init_scaling(const obs_block_type *obs_block,
     *__obs_offset = obs_offset;
 }
 
-/*
```

Comment:
Looks like this could just be removed.

Reply:
Not sure what you are pointing to here.
Line 212 in `obs_data.cpp` is the definition of `obs_block_initObs` on my end.
Is that what you were thinking about?

Reply:
To me it looked like you had commented out a static function, and if that worked OK I guess the function could just be removed. I have probably misread the diff.